src/components/20-molecules/policy-features/index.js

```diff
@@ -0,0 +1,62 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+
+/* eslint-disable import/no-extraneous-dependencies */
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+import './policy-features-item/index';
+
+export const STYLE_WHITELIST = [
+  'dark-indigo',
+  'axa-blue',
+  'wild-sand',
+  'white',
+];
+
+class AXAPolicyFeatures extends LitElement {
+  static get tagName() {
+    return 'axa-policy-features';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      title: { type: String },
+      variant: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.variant = 'dark-indigo';
```

Comment:
I would like more to see `dark indigo` as default styling. Because in best case the user have nothing to configure and we don't have to preselect something that is default.